From 3632a90fbbed8a584fab2bad4eef90cfc284569a Mon Sep 17 00:00:00 2001
From: Michael D Labriola <mdlabriola@yahoo.com>
Date: Sat, 14 Aug 2010 09:43:54 -0400
Subject: [PATCH 2/3] more defines for bootstrap builds

This commit adds a couple defines to linux.h to make sure GCC is looking in
/bootstrap/include and /bootstrap/lib
---
 gcc/config/linux.h |   11 +++++++++++
 1 files changed, 11 insertions(+), 0 deletions(-)

diff --git a/gcc/config/linux.h b/gcc/config/linux.h
index 7908af7..fc810b4 100644
--- a/gcc/config/linux.h
+++ b/gcc/config/linux.h
@@ -135,3 +135,14 @@ see the files COPYING3 and COPYING.RUNTIME respectively.  If not, see
 #define TARGET_HAS_SINCOS (OPTION_GLIBC)
 
 #define TARGET_POSIX_IO
+
+/* the following defines are for the Source Ruckus bootstrap build stages */
+
+#undef STANDARD_INCLUDE_DIR
+#define STANDARD_INCLUDE_DIR "/bootstrap/include/"
+
+#undef STANDARD_STARTFILE_PREFIX_1
+#define STANDARD_STARTFILE_PREFIX_1 "/bootstrap/lib/"
+
+#undef STANDARD_STARTFILE_PREFIX_2
+#define STANDARD_STARTFILE_PREFIX_2 ""
-- 
1.7.1.1

